\RequirePackage{nag}
\documentclass[a0,portrait]{a0poster}

\usepackage{klmrposter}

\begin{document}
\sffamily

\noindent
\begin{minipage}[][10cm][t]{0.88\textwidth}
    {
        \VeryHuge\color{accent}\bfseries\sffamily%
        Codon–anticodon interactions in mammals do not adapt to changes in
        cellular function
    }
    {
        \\[\baselineskip]
        \noindent
        \Large\color{Black}%
        \underline{Konrad L.\ M.\ Rudolph\textsuperscript{1}},
        Bianca M.\ Schmitt\textsuperscript{2},
        Duncan T.\ Odom\textsuperscript{2},
        John C.\ Marioni\textsuperscript{1},
        Claudia Kutter\textsuperscript{2}
    }
\end{minipage}%
\hfill%
\begin{minipage}[][10cm][t]{0.1\textwidth}
    \raggedleft
    \includegraphics[height=10cm]{portrait}
\end{minipage}

\vspace{3cm}
\Large

\noindent
\begin{minipage}[t][][t]{0.45\textwidth}
    \section*{Introduction}
    We have previously reported the stability of the codon--anticodon interface
    during mouse development, where we measured the whole-transcriptome codon
    usage and the tRNA anticodon abundance of matching stages of development and
    found remarkable stability, both in codon usage and in tRNA anticodon
    abundance, despite the drastic phenotypic changes \citep{Schmitt:2014}.
    These findings let us to hypothesise that, unlike in other organisms
    \citep{Carlini:2003}, no widespread translational regulation, regulated by
    codon bias, is likely to occur in mammals.

    In contrast, others have recently reported that codon usage for genes
    enriched in cell-autonomous function and multicelluarity in humans is
    adapted to the tRNA abundance of proliferating and differentiating cells,
    respectively \citep{Gingold:2014}.

    Here we test this hypothesis using whole-genome expression data from healthy
    liver tissue and two different tumour cell lines in human and mouse.
\end{minipage}%
\hfill%
\begin{minipage}[t][][t]{0.5\textwidth}
    \section*{Codon usage bias correlates with gene set size}
    \includegraphics[width=\textwidth]{gene-set-size}
\end{minipage}

\begin{minipage}[][][t]{\textwidth}
    \section*{Experimental setup}
    \begin{minipage}[c]{0.50\textwidth}
        \includegraphics[width=\textwidth]{experimental-design}
        \\[1cm]
        Codon usage and Anticodon abundance of healthy adult liver and two
        cancer cell lines were correlated to estimate their adaptation in each
        cellular condition. Higher correlation implies better adaptation of the
        codon usage to the anticodon abundance.

        The GO terms for cell-specific functions are “pattern specification
        process” and “M phase of mitotic cell cycle” for healthy and tumour
        cells, respectively.
    \end{minipage}%
    \hfill
    \begin{minipage}{0.20\textwidth}
        \centering
        \textbf{Legend:}\\[0.5cm]
        \includegraphics[width=\textwidth]{legend-1}
        \\[2cm]
        \includegraphics[width=\textwidth]{legend-2}
    \end{minipage}%
    \hfill
    \begin{minipage}{0.20\textwidth}
        \centering
        \includegraphics[height=15cm]{interaction-schema}
    \end{minipage}%
\end{minipage}

\vspace{2cm}
\begin{minipage}{\textwidth}
    \begin{minipage}[t]{0.48\textwidth}
        \section*{Results}
        \includegraphics[width=\textwidth]{results}
    \end{minipage}%
    \hfill%
    \begin{minipage}[t][][t]{0.48\textwidth}
        \LARGE
        \section*{Tumour cell types show no evidence of codon--anticodon coordination}
        Codon--anticodon correlation is not significantly higher in matching
        than in mismatching cellular condition. The same is true in genes in
        condition-specific gene sets (determined either via differential
        expression or Gene Ontology). Gene sets determined from Gene Ontology,
        on the contrary, show significantly less codon--anticodon correlation,
        consistent with the stochastic effect of small gene set sample sizes
        (\(N<100\) in all cases) on codon bias.

        \vspace{7cm}
        \hfill Contact: rudolph@ebi.ac.uk, \textbf{@klmr}
    \end{minipage}%
\end{minipage}

\vspace{2cm}
\section*{References}
\vspace{-1cm}
\begin{multicols}{2}
    \renewcommand\section[2]{}
    \bibliography{references.bib}
\end{multicols}

\vfill
\noindent
\begin{center}
\normalsize%
1. European Molecular Biology Laboratory, European Bioinformatics Institute •
2. University of Cambridge, Cancer Research UK Cambridge Institute
\end{center}
\vspace{1cm}

\noindent
\begin{minipage}[][][b]{0.3\textwidth}
    \centering
    \includegraphics[height=4cm]{Colour_logo_pantone_DM}
\end{minipage}%
\hfill%
\begin{minipage}[][][b]{0.3\textwidth}
    \centering
    \includegraphics[height=4cm]{cruk-logo}
\end{minipage}%
\hfill%
\begin{minipage}[][][b]{0.3\textwidth}
    \centering
    \includegraphics[height=4cm]{EMBL-EBIlogo}
\end{minipage}%
\end{document}
